src/redux/actions/pipeline/runGem2s.js

```diff
@@ -47,6 +45,8 @@ const runGem2s = (experimentId) => async (dispatch, getState) => {
 
     dispatch(loadBackendStatus(experimentId));
 
+    const projectId = experiments[experimentId].projectUuid;
```

Comment:
this fixes the bug where 'relaunch current analysis' does not work and throws a 500